test/end-to-end/pages/screeners/willleft.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/willleft');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
+const willLeftEn = require('app/resources/en/translation/screeners/willleft');
+const willLeftCy = require('app/resources/cy/translation/screeners/willleft');
 const config = require('config');
+const pageUnderTest = require('app/steps/ui/screeners/willleft');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?